style.css

```diff
@@ -3,9 +3,12 @@ body {
     margin: 0;
     background-color: hsl(185, 41%, 84%);
     font-family: 'Space Mono', monospace;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+    height: 100vh;
 }
-
-
 .input {
     position: relative;
 }
```

Comment:
Class name input with a single property?
The naming convention can be used to specify what input you want to apply this relative property to.
We do have an input selector.

Reply:
https://github.com/sahibkaur/tip-calculator/pull/3/commits/2a9b11befd879239eb4582fc6bcc8fe6e6f60da9